R/detect_column.r

```diff
@@ -0,0 +1,354 @@
+library(rio)
```

Comment:
Minor comment for consistency:  Please change the file extension to an upper case R.